Test: rebuild disks from bootable containers when the BIB container changes

achilleas-k opened the pull request:
Write the [bootc-image-builder](https://quay.io/repository/centos-bootc/bootc-image-builder) container ID that we used to build a disk image and boot test bootable containers.  We use this ID to retest bootable container images when the bib container changes to ensure we maintain compatibility.

ondrejbudai reviewed (dismissed):
Yay!

achilleas-k commented:
Requested review from @mvo5 because tests ^_^

achilleas-k commented:
Okay, the bib test failed.  This is good :)

mvo5 commented on test/scripts/test_imgtestlib.py:
```diff
@@ -21,3 +33,49 @@ def test_read_seed():
+}
+
+
+@pytest.mark.parametrize("arch", ["amd64", "arm64"])
```
(nipitck) given that `["amd64", "arm64"]` is repeated maybe `TEST_ARCHES = ["amd64", "arm64"]` or something? Then if we add s390x or riscv64 we only need to change one place?

achilleas-k commented:
Current status: The image doesn't build with BIB since we switched to using bootupd.  The metadata isn't part of our base image.  This was expected (and the reason I added the test in the first place).  Working on adding this now.

achilleas-k commented:
Rebased.  Should be good to go!

ondrejbudai commented on test/scripts/test_imgtestlib.py:
```diff
@@ -32,3 +46,48 @@ def test_path_generators():
+    transport = "containers-storage:"
+    image = "registry.gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/manifest-list-test:latest"
+    with tempfile.TemporaryDirectory() as tmpdir:
+        testlib.runcmd(["sudo", "podman", "pull", f"--arch={arch}", "--storage-driver=vfs", f"--root={tmpdir}", image])
```
`--storage-driver=vfs` is so we can run the test in a container, right? It might be worth leaving a small note about that here, it might not be obvious to future readers.

EDIT from future Ondrej who read all commits: So it apparently doesn't work in a container... weird...

ondrejbudai reviewed (approved):
Awesome! I think we can resolve my comment in a follow-up, the test is disabled nevertheless.
